src/gowrapper/ckks/keygen.go

```diff
@@ -184,6 +199,75 @@ func lattigo_genRotationKeysForRotations(keygenHandle Handle5, skHandle Handle5,
 	return marshal.CrossLangObjMap.Add(unsafe.Pointer(rotKeys))
 }
 
+//export lattigo_getRotationKey
+func lattigo_getRotationKey(paramHandle, rotKeysHandle Handle5, rotationStep int) Handle5 {
```

Comment:
replace int with uint64_t (@crockeea requested to use it https://github.com/awslabs/aws-cppwrapper-lattigo/pull/15#discussion_r1112363018)